### PR TITLE
Adding textarea

### PR DESCRIPTION
**Pull request by marciovicente**
image

### CODE REVIEW

File: dist/css/general.css [hunk 1]
```diff
@@ -3862,6 +3862,12 @@ label.switch input[type="radio"][disabled] ~ .flatButton--red.switch-label {
   padding: 0 24px 0 8px;
   /* 1 */
 }
+.field--small textarea {
+  height: 80px;
```
Comment (andersonba) [inline dist/css/general.css]: Use $grid

File: scss/components/_inputs.scss [hunk 2]
```diff
@@ -48,6 +53,12 @@ $field-horizontalPadding--small:  $grid * 1;
     font-size: 1.3rem;
     padding: 0 ($field-horizontalPadding--small + $grid * 2) 0 $field-horizontalPadding--small; /* 1 */
   }
+  textarea {
+    height: $textarea-height--small;
+    padding: $field-horizontalPadding--small;
+    font-size: 1.3em;
```
Comment (andersonba) [inline scss/components/_inputs.scss]: Use `rem` instead `em`

Comment (marciovicente) [inline scss/components/_inputs.scss]: :+1:

Comment (shankarcabus) [inline scss/components/_inputs.scss]: Yeah!

File: scss/components/_inputs.scss [hunk 3]
```diff
@@ -48,6 +53,12 @@ $field-horizontalPadding--small:  $grid * 1;
     font-size: 1.3rem;
     padding: 0 ($field-horizontalPadding--small + $grid * 2) 0 $field-horizontalPadding--small; /* 1 */
   }
+  textarea {
+    height: $textarea-height--small;
+    padding: $field-horizontalPadding--small;
+    font-size: 1.3rem;
```
Comment (shankarcabus) [inline scss/components/_inputs.scss]: The `.field` already has `font-size: 1.3rem`

**Comment by marciovicente** [comment]
Done @shankarcabus @andersonba

**Comment by andersonba** [comment]
LGTM

**Pull request merged**